README.md

```diff
@@ -1,2 +1,58 @@
-# jupyterhub-ssh
-SSH Access to JupyterHubs
+# JupyterHubSSH
+
+[![Documentation build status](https://img.shields.io/readthedocs/jupyterhub?logo=read-the-docs)](https://jupyterhub-ssh.readthedocs.io/en/latest/)
+
+SSH Access to JupyterHubs, independent of the way there were deployed.
+
+## Development Status
+This project is under active develpoment :tada:, so expect a few changes along the way.
+
+## Technical Overview
+
+JupyterHubSSH is made up of two main components:
+
+- a Notebook SSH server that maps a SSH connection to a Notebook server on a JupyterHub.
+- a [Terminado](https://github.com/jupyter/terminado) client that knows how to connect and communicate to a Jupyter terminal.
+
+![Overview](https://raw.githubusercontent.com/yuvipanda/jupyterhub-ssh/main/docs/source/_static/images/technical-overview.png)
+
```

Comment:
Some note about the role of the helm-chart in this repo would be nice to have as part of this section.

Reply:
To be honest, I missed the helm-chart part in the codebase :sweat_smile: I'll look into it and add it here once I get an understanding of it :D. Thanks @consideRatio :heart: